pkg/binaries/tasks.go

```diff
@@ -135,3 +135,21 @@ func (a *ArtifactDownload) Execute(runtime connector.Runtime) error {
 	}
 	return nil
 }
+
+type RegistryPackageDownload struct {
+	common.KubeAction
+}
+
+func (k *RegistryPackageDownload) Execute(runtime connector.Runtime) error {
+	arch := runtime.GetHostsByRole(common.Registry)[0].GetArch()
+
+	packageDir := filepath.Join(runtime.GetWorkDir(), arch)
```

Comment:
How do you think modifying the path as `kubekey/registry/amd64/xxxx.tar.gz`? That will be the same as the kk k8s binaries path(`kubekey/v1.21.5/amd64/xxxx.tar.gz`).

Reply:
It is not recommended to put it in the k8s version directory. If there are multiple versions in the offline package, the files will be saved repeatedly.

Reply:
I think these files (helm / etcd / cni / docker / crictl ...) should also be stored in a separate directory.

Reply:
> It is not recommended to put it in the k8s version directory. If there are multiple versions in the offline package, the files will be saved repeatedly.

I mean put it in a dir which named `registry` not in the k8s version dir.

Reply:
Done